Leaflet.Graticule.js

```diff
@@ -187,43 +189,102 @@ L.LatLngGraticule = L.Layer.extend({
         L.DomUtil.setOpacity(this._canvas, this.options.opacity);
     },
 
+    // todo: Decimal degrees to Degrees - Decimal Minutes
+    __deg_to_dmd: function(deg) {
+
+        var d = Math.floor (deg);
```

Comment:
What does `d` stand for? Please be more explicit with this variable name.

Reply:
d means degrees. I'll will change it to `degrees`